app/views/admin/transactions/index.html.slim

```diff
@@ -0,0 +1,6 @@
+- content_for(:title, 'Transactions')
+
+table.table
+  = render 'thead'
+  tbody
+    = render 'tbody'
```

Comment:
This seems a bit premature to separate the head and body. Each partial also adds a bit to the load time I find. Can we combine these back into one file unless there is some need for the abstraction?